apollo/src/graphql/model-version/modelVersionMutations.ts

```diff
@@ -68,6 +69,9 @@ builder.mutationFields((t) => ({
         },
         async resolve(root, args, ctx) {
             const results = ObjectionMLModelVersion.transaction(async (trx) => {
+                const user = (await ObjectionUser.query()
```

Comment:
Next PR I might just move this into `src/index` where I'm building the context object so we don't have to keep including this boilerplate in every single mutation